### PR TITLE
Add some smool stuff

### PR DESCRIPTION
**Pull request by Adrian8115**
- Change stuff in `Mesh.cpp` and `Mesh.hpp` for the sake of consistency
- Add `ChunkState` enum

### CODE REVIEW

File: AmethystAPI/src/minecraft/src/common/world/level/ChunkState.hpp
Comment (FrederoxDev) [inline AmethystAPI/src/minecraft/src/common/world/level/ChunkState.hpp:1]: OCD thing, this should be at the path: `src/common/world/level/chunk/ChunkState.hpp`, ik you have no way to know this xD

**Review by FrederoxDev — COMMENTED** [review]
Thanks! Just uno simple file move and good to go!

File: AmethystAPI/src/minecraft/src/common/world/level/ChunkState.hpp
Comment (Adrian8115) [reply AmethystAPI/src/minecraft/src/common/world/level/ChunkState.hpp:1]: how do you even know this?

Comment (FrederoxDev) [reply AmethystAPI/src/minecraft/src/common/world/level/ChunkState.hpp:1]: ill tell u in discord

Comment (Adrian8115) [reply AmethystAPI/src/minecraft/src/common/world/level/ChunkState.hpp:1]: logical, my bad... I will look for them in the future

File: AmethystAPI/src/minecraft/src/common/world/level/chunk/ChunkState.hpp
Comment (FrederoxDev) [inline AmethystAPI/src/minecraft/src/common/world/level/chunk/ChunkState.hpp:1]: yippee

**Review by FrederoxDev — APPROVED** [review]

File: AmethystAPI/src/minecraft/src/common/world/level/chunk/ChunkState.hpp
Comment (Adrian8115) [reply AmethystAPI/src/minecraft/src/common/world/level/chunk/ChunkState.hpp:1]: https://github.com/FrederoxDev/Amethyst/assets/127942625/3c9a311d-7283-43bc-ae05-a4ad1dfd841e

**Pull request merged**